content/docs/command-reference/checkout.md

```diff
@@ -12,7 +12,7 @@ usage: dvc checkout [-h] [-q | -v] [--summary] [-d] [-R] [-f]
 
 positional arguments:
   targets       Limit command scope to these tracked files/directories,
-                .dvc files, or stage names.
+                .dvc files and stage or foreach-group names.
```

Comment:
I am not sure if people would know what `foreach group` means. 

It is not used in https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#foreach-stages which are the only existing docs for `foreach`

Reply:
Agreed. I don't think this change is needed TBH: foreach groups ultimately become stages so it's covered with "or stage names". No need to be that super explicit about every possible syntax in the help output, especially for such an advanced feature (not that used from what I understand) -- explaining that you can target them in the foreach docs should be enough IMO.